### PR TITLE
[Element 2.0] Create consistency between browser's actions of script and reporter

### PR DESCRIPTION
**Pull request by honglax**
I've created new functions without the decorator, and call it from the original function. I also edited the decorator `autoWaitUntil`.

### CODE REVIEW

File: packages/core/src/runtime/Browser.ts [hunk 1]
```diff
@@ -607,4 +567,58 @@ export class Browser<T> implements BrowserInterface {
...
+		const locator = locatableToLocator(locatable, 'browser.findElement(locatable)')
+
+		debug('locator %o', locator)
+
+		const maybeElement = await locator.find(this.page, undefined, this.target)
+		if (!maybeElement) {
+			throw toLocatorError(locatable, 'browser.findElement()')
+		}
+		const element = maybeElement as ElementHandle
+
```
Comment (phileba) [inline packages/core/src/runtime/Browser.ts:585]: should remove this empty line

Comment (honglax) [reply packages/core/src/runtime/Browser.ts:585]: Got it.

Comment (honglax) [reply packages/core/src/runtime/Browser.ts]: Hi a @phileba, please check the change and let me know if you have any concern. Thank you.

Comment (phileba) [reply packages/core/src/runtime/Browser.ts]: this is small, you can change and mark resolved by yourself.

**Review by phileba — APPROVED** [review]

**Pull request merged**